docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/MulticlassClassification/MulticlassClassification.ttinclude

```diff
@@ -75,7 +75,7 @@ namespace Samples.Dynamic.Trainers.MulticlassClassification
         private static IEnumerable<DataPoint> GenerateRandomDataPoints(int count, int seed=0)
         {
             var random = new Random(seed);
-            float randomFloat() => (float)random.NextDouble();
+            float randomFloat() => (float)(random.NextDouble() - 0.5);
```

Comment:
>float randomFloat() => (float)(random.NextDouble() - 0.5); [](start = 12, length = 58)

This is great, but did you re-generate all the TT by running custom tool to make sure the samples are not broken?

Reply:
yes, only 3 tt depend on this one, they are regenerated


---
In reply to: [274642625](https://github.com/dotnet/machinelearning/pull/3246#discussion_r274642625) [](ancestors = 274642625)